velodrome/grafana-stack/influxdb.yaml

```diff
@@ -22,7 +22,10 @@ spec:
               name: influxdb
               key: rootpassword
         - name: PRE_CREATE_DB
-          value: github
+          value: github;monitoring
```

Comment:
I'm glad you ask!
Sure of course, it's not just random :-)

https://github.com/tutumcloud/influxdb/blob/master/0.13/run.sh#L100

This variable is used to create the table on the initialization (first run) of the influxdb container.